src/views/MainPage/MainPage.js

```diff
@@ -65,21 +75,21 @@ function renderMainView() {
   const creditsButton = Button('Credits', 'credits-button', true, 'click', handleNavigationButtonClick);
 
   navContainer.append(quizButton, leaderboardButton, adoptionButton, creditsButton);
-  const navItems = navContainer.querySelectorAll('a');
-
-  navItems.forEach((item) => {
-    item.addEventListener('click', () => {
-      alert(`Hello! I'm ${item.dataset.name} site`);
-    });
-  });
 }
 
-async function renderAnimalFact() {
-  const factItem = document.querySelector('.bubble-text');
+function renderAnimalFact() {
+  const factSentenceItem = document.querySelector('.bubble.fact .bubble-text');
   const factSentence = document.createElement('p');
-  const FACT =
-    "Cat's eyes shine in the dark because of the tapetum, a reflective layer in the eye, which acts like a mirror.";
 
-  factSentence.innerText = FACT;
-  factItem.append(factSentence);
+  retrieveAnimalFact()
+    .then((result) => {
+      const newFact = new Fact(result);
+      factSentence.innerText = newFact.makeFact;
+    })
+    .catch((error) => {
+      const newFact = new Fact(FACT_ANIMAL.fallbackFact);
+      factSentence.innerText = newFact.makeFact;
+    });
```

Comment:
Per our conversation on discord - Fact object is useless, because:
* it holds only one string
* it is created here and get's removed (because of the block scope)
* it will be problematic to extend fact with something more (how about extending Fact class with random image)

Create Fact with default message in the main render function here, use setter to change message when random fact is retrieved, and render the message based on Fact object. It will be easier to use more APIs ex. to add random image to the Fact